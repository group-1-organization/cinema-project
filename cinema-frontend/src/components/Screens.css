.header {
  padding: 50px;
  text-align: center;
  background: #113a80;
  color: white;
  font-size: 30px;
}