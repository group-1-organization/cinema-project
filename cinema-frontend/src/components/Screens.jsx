import { Card, Container } from 'react-bootstrap';
import { Button } from 'react-bootstrap';
import { Collapse } from 'react-bootstrap';
import { useState } from 'react';

import StandardSeating from '../standardseating.png'
import StandardDecor from '../standarddecor.jpg'
import PremiumDecor from '../premiumdecor.jpg'
import PremiumSeating from '../deluxeseating.PNG'
import '../App.css'

const Screens = () => {
    const [open, setOpen] = useState(false);

    return (
        <div>
            <div class="page-header">
                <h1>Screens</h1>
            </div>
            <br />
            <div className="black-font">
                <Container>
                    {/* <Card className="text-center">
                        <Card.Header>
                            <h2>Standard screens</h2>
                        </Card.Header>
                        <Card.Body>
                            <Card.Text>
                                Our standard screens feature comfortable seating with an arm rest on one side featuring a cup holder, with appreciable leg room and a great soundscape.

                            </Card.Text>
                            <br />
                            <img src={StandardDecor} alt='Standard decor' />
                            <br />
                            <Collapse in={open}>
                                <Button
                                    onClick={() => setOpen(!open)}
                                    aria-controls="collapse-standard-plan"
                                    aria-expanded={open}
                                >
                                    View Standard screen seating plan
                                </Button>
                                <div id="collapse-standard-plan">
                                    <img src={StandardSeating} alt='Standard seating' />
                                </div>
                            </Collapse>
                        </Card.Body>

                    </Card> */}

                    <h2>Standard screens</h2>
                    <div>
                        <p>Our standard screens feature a typical seating plan with ample leg room, and an excellent soundscape</p>
                        <center>
                            <img src={StandardSeating} alt='Standard seating' />
                        </center>
                    </div>
                    <br />
                    <div>
                        <p>Our standard screens feature comfortable seating with an arm rest on one side featuring a cup holder.</p>
                        <center>
                            <img src={StandardDecor} alt='Standard decor' />
                        </center>
                    </div>
                    <br />
                    <h2>Premium screens</h2>
                    <div>
                        <p>Our premium screens feature a typical seating plan with significant leg room, an excellent soundscape, and a smaller room for a more initimate feel.</p>
                        <center>
                            <img src={PremiumSeating} alt='Premium seating' />
                        </center>
                    </div>
                    <br />
                    <div>
                        <p>Our premium screens feature extra comfortable seating with reclining chairs, an arm rest on each side both featuring a cup holder, and cushions for extra support.</p>
                        <center>
                            <img src={PremiumDecor} alt='Premium decor' />
                        </center>
                    </div>
                    <br />
                </Container>
            </div>
        </div>
    );
}

export default Screens;