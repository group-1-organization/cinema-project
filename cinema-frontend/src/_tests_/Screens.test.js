import renderer, { act } from 'react-test-renderer';
import Screens from '../components/Screens';

test('Screens page matches snapshot', () => {
    const tree = renderer.create(<Screens />);

    expect(tree).toMatchSnapshot();
});