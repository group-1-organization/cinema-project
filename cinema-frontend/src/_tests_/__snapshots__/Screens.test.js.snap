// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens page matches snapshot 1`] = `
<div>
  <div
    class="page-header"
  >
    <h1>
      Screens
    </h1>
  </div>
  <br />
  <div
    className="black-font"
  >
    <div
      className="container"
    >
      <div
        className="screen-card card"
      >
        <div
          className="card-header"
        >
          <h2>
            Standard screens
          </h2>
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Our standard screens feature comfortable seating with an arm rest on one side featuring a cup holder, with appreciable leg room and a great soundscape.
          </p>
          <br />
          <img
            alt="Standard decor"
            src="standarddecor.jpg"
          />
          <br />
          <br />
          <button
            aria-controls="collapse-standard-plan"
            aria-expanded={false}
            className="button-colour btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            View Standard screen seating plan
          </button>
          <br />
          <br />
          <div
            aria-expanded={null}
            className="collapse"
            id="collapse-standard-plan"
          >
            <img
              alt="Standard seating"
              src="standardseating.png"
            />
          </div>
        </div>
      </div>
      <br />
      <div
        className="screen-card card"
      >
        <div
          className="card-header"
        >
          <h2>
            Premium screens
          </h2>
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Our premium screens feature extra comfortable seating with reclining chairs, an arm rest on each side both featuring a cup holder, and cushions for extra support. They also feature significant leg room, an excellent soundscape, and a smaller room for a more initimate feel.
          </p>
          <br />
          <img
            alt="Premium decor"
            src="premiumdecor.jpg"
          />
          <br />
          <br />
          <button
            aria-controls="collapse-premium-plan"
            aria-expanded={false}
            className="button-colour btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            View Premium screen seating plan
          </button>
          <br />
          <br />
          <div
            aria-expanded={null}
            className="collapse"
            id="collapse-premium-plan"
          >
            <img
              alt="Premium seating"
              src="deluxeseating.PNG"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
